docs/documentation/siddhi-4.0.md

```diff
@@ -104,63 +101,66 @@ The source syntax is as follows:
 )
 define stream stream_name (attribute1 Type1, attributeN TypeN);
 ```
-
+This syntax includes the following annotations.
 **Source**
 
 The `type` parameter of `@source` defines the source type that receives events. The other parameters to be configured 
-depends on the source selected, some of those can also be optional values. 
-For detailed information about the parameters refer the appropriate source documentation.
-
-Some of the supports source types are:
-
-* <a target="_blank" href="https://wso2-extensions.github.io/siddhi-io-tcp/">TCP</a>
```

Comment:
please use <a target="_blank" so that page will be redirected

Reply:
Done